src/plugins/drawing/objects/group.tsx

```diff
@@ -48,10 +48,14 @@ export const GroupObject = DrawingObject.named("GroupObject")
       return "Group";
     },
     get icon() {
-      return GroupObjectsIcon;
+      return (<GroupObjectsIcon viewBox="0 0 36 34"/>);
```

Comment:
Given that a bunch of icons have `viewBox="0 0 36 34"`, might make sense to make the dimensions a constant.